client/pipeline_events_test.go

```diff
@@ -0,0 +1,61 @@
+package client_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/calyptia/api/types"
+)
+
+func TestClient_CreatePipelineEvents(t *testing.T) {
```

Comment:
These tests will fail if we merge this. Can you comment them out, and we run them once we implement cloud.

Reply:
Will a quick return work just as well?

Reply:
I think. Maybe a t.SkipNow could work too.